kore/app/exec/Main.hs

```diff
@@ -349,13 +342,13 @@ parseKoreExecOptions startTime =
         option (readSum "strategy" strategies)
             (  metavar "STRATEGY"
             <> long "strategy"
-            <> value ("all", priorityAllStrategy)
+            <> value ("all", All)
             <> help "Select rewrites using STRATEGY."
             )
       where
         strategies =
-            [ ("any", priorityAnyStrategy)
-            , ("all", priorityAllStrategy)
+            [ ("any", Any)
+            , ("all", All)
```

Comment:
Because this uses an enumeration type now, there is no reason to keep the `String` part of the tuple.